go/simple/v1/ampli/ampli.go

```diff
@@ -0,0 +1,385 @@
+// ampli.py
+//
+// Ampli - A strong typed wrapper for your Analytics
+//
+// This file is generated by Amplitude.
+// To update run 'ampli pull go-ampli'
+//
+// Required dependencies: analytics-go
+// Tracking Plan Version: 0
+// Build: 1.0.0
+// Runtime: go-ampli
+//
+// [View Tracking Plan]: https://data.amplitude.com/test-codegen/Test%20Codegen/events/main/latest
+//
+// [Full Setup Instructions]: https://data.amplitude.com/test-codegen/Test%20Codegen/implementation/main/latest/getting-started/go-ampli
+//
+
+package ampli
+
+import (
+	"log"
+
+	"github.com/amplitude/analytics-go/amplitude"
+)
+
+var Client = Ampli{}
+
+type Environment string
+
+const (
+	DEVELOPMENT Environment = "development"
+	PRODUCTION  Environment = "production"
+)
+
+var APIKey = map[Environment]string{
+	DEVELOPMENT: "",
+	PRODUCTION:  "",
+}
+
+var DefaultConfiguration = amplitude.Config{
+	FlushInterval:   amplitude.DefaultFlushInterval,
```

Comment:
All members (except `Plan`) should be initialized in the core SDK

Reply:
I used all default values in core SDK. But it definitely makes it more readable to use `NewConfig` function from Core SDK here.

Updated. Use a `DefaultConfiguration()` func to return default config with default plan initialized